config/mcu_definitions/BTT_SKR_E3mini_v3.cfg

```diff
@@ -0,0 +1,24 @@
+[board_pins mcu_manufacturer]
+aliases:
+    MCU_XM_STEP=PB13 , MCU_XM_DIR=PB12 , MCU_XM_ENABLE=PB14 ,
+    MCU_YM_STEP=PB10 , MCU_YM_DIR=PB2  , MCU_YM_ENABLE=PB11 ,
+    MCU_ZM_STEP=PB0  , MCU_ZM_DIR=PC5  , MCU_ZM_ENABLE=PB1  ,
+    MCU_EM_STEP=PB3  , MCU_EM_DIR=PB4  , MCU_EM_ENABLE=PD1  ,
+    MCU_TMCUART=PC11 , MCU_TMCTX=PC10  ,
```

Comment:
MCU_TMCUART and MCU_TMCTX are configured here for all axis. Same pin

Reply:
Hi, I think this is correct and same as how the SKR Mini E3 v2 handles it.  All of the drivers are tied to the same uart line I think

Reply:
i commented it as reference to my other comment :)

Reply:
resolved per other discussions